### PR TITLE
Make author avatars meet a11y text contrast on dashboard

### PR DESCRIPTION
**Pull request by Piinks**
This change makes the text contrast of the author avatars meet a11y guidelines. 

Prior to change:

Screen Shot 2020-07-16 at 11 39 54 AM

With change (dark and light themes):

Screen Shot 2020-07-16 at 12 05 24 PM

Screen Shot 2020-07-16 at 12 30 42 PM

### CODE REVIEW

**Comment by Piinks** [comment]
@keyonghan it appears here that unrelated files in app_flutter are failing due to formatting.

**Comment by Piinks** [comment]
Oh I think I see. Let me check the 120 line length for app_flutter

File: app_flutter/test/widgets/accessibility_test.dart [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:app_flutter/widgets/commit_author_avatar.dart';
+import 'package:cocoon_service/protos.dart' show Commit;
+
+void main() {
+  group('Author avatars meet guidelines for theme brightness', () {
+    Widget buildAuthors() {
+      final List<String> names = <String>[
```
Comment (digiter) [inline app_flutter/test/widgets/accessibility_test.dart:14]: Use a for loop to generate this?

File: app_flutter/lib/widgets/commit_author_avatar.dart [hunk 2]
```diff
@@ -24,21 +24,23 @@ class CommitAuthorAvatar extends StatelessWidget {
...
-    if (Theme.of(context).brightness == Brightness.dark) {
-      authorColor = HSLColor.fromColor(authorColor).withLightness(.75).toColor();
-    }
+    final int authorHash = authorName.hashCode;
+    final ThemeData theme = Theme.of(context);
+
+    final double hue = (360.0 * authorHash / (1 << 15)) % 360.0;
+    final double themeValue = HSVColor.fromColor(theme.backgroundColor).value;
+    Color authorColor = HSVColor.fromAHSV(1.0, hue, 0.4, themeValue).toColor();
+    if (theme.brightness == Brightness.dark) authorColor = HSLColor.fromColor(authorColor).withLightness(.65).toColor();
```
Comment (keyonghan) [inline app_flutter/lib/widgets/commit_author_avatar.dart:34]: This line seems to fail the analyzer:
```
Analyzing app_flutter...                                        

   info â€¢ Separate the control structure expression from its statement â€¢ lib/widgets/commit_author_avatar.dart:34:46 â€¢ always_put_control_body_on_new_line

1 issue found. (ran in 5.1s)
```

Comment (Piinks) [reply app_flutter/lib/widgets/commit_author_avatar.dart:34]: Oh man I missed that one, thanks! dartfmt makes this a one-liner, but the analyzer fails it, so you have to put brackets around single lined statements to pass. 🤦 
Maybe we can unify the formatting in this repo. I spent most of my time on this change trying to navigate the eccentricities of getting the pre-submit checks to pass the formatting.

**Comment by keyonghan** [comment]
Did you get any chance to rebase with master? @CaseyHillers pushed some changes to fix the app_flutter test issues days ago.

**Comment by Piinks** [comment]
> Did you get any chance to rebase with master? @CaseyHillers pushed some changes to fix the app_flutter test issues days ago.

I just branched off master yesterday. I'll update again to be sure...

**Comment by keyonghan** [comment]
> > Did you get any chance to rebase with master? @CaseyHillers pushed some changes to fix the app_flutter test issues days ago.
> 
> I just branched off master yesterday. I'll update again to be sure...

Then you should have already picked up the changes.. If this issue persists, @CaseyHillers may help.

**Comment by Piinks** [comment]
> > > Did you get any chance to rebase with master? @CaseyHillers pushed some changes to fix the app_flutter test issues days ago.
> > 
> > 
> > I just branched off master yesterday. I'll update again to be sure...
> 
> Then you should have already picked up the changes.. If this issue persists, @CaseyHillers may help.

Ah, the failures are valid. The goldens need to be updated, but I don't have access to a linux machine.

**Comment by CaseyHillers** [comment]
Updated 🤞

**Comment by Piinks** [comment]
Thank you!

**Pull request merged**